src/app/app.module.ts

```diff
@@ -53,6 +53,7 @@ import { AddRegressionRegionModal } from './shared/components/addregressionregio
 import { GagestatsComponent } from './gagestats/gagestats.component';
 import { GsSidebarComponent } from './gagestats/gs-sidebar/gs-sidebar.component';
 import { GagestatsService } from './gagestats/gagestats.service';
+import { AddStationModal } from './shared/components/add-station/add-station.component';
```

Comment:
We could maybe move this into the `gagestats` folder instead, since this will only be used in the gagestats page?